### DISCUSSION TITLE
Not an issue: thanks for sharing your code :3

### DISCUSSION THREAD

**Discussion by LittleFox94** [justinian/jsix#6] [General]
Just wanted to thank you for sharing your code, helped me figure out how to use PAT and use it in my own OS - which can now run Doom thanks to you :D

**Comment by justinian** [comment]
That's awesome! Thanks for letting me know, that made my day. 😁 Maybe someday I'll hit that Doom milestone too.

**Comment by LittleFox94** [comment]
Since you appreciate me telling you that, have a link to the video where I show it:  
https://twitter.com/0x0a_fox/status/1386095877165551616

Didn't want to add a link from the beginning to not have it look like spam x)

**Comment by LittleFox94** [comment]
Also, doom is kinda easy to port ^^

You need a framebuffer, some libc things, a way to get input into it and a way to load files (or compile them into the binary like I cheated xD). Will push my changes to Github soon if you want to look at that ^^

**Comment by justinian** [comment]
Yeah I'd love to see it!

**Comment by LittleFox94** [comment]
it's here: https://github.com/LittleFox94/fbDOOM

Going through history might be easist to understand my changes, tried to have reasonable commit messages and bundle changes together
